lab-matt/route/talent-router.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const debug = require('debug')('auth:talent-router');
+const Router = require('express').Router;
+const jsonParser = require('body-parser').json();
+const httpErrors = require('http-errors');
+
+const parseBearerAuth = require('../lib/parse-bearer-auth');
+const talentController = require('../controller/talent-controller');
+
+
+const talentRouter = module.exports = new Router();
+
+talentRouter.post('/talent', parseBearerAuth, jsonParser, function(req, res, next){
```

Comment:
I feel like this router could be cleaned up a bit.
Declaring a route and then chaining methods can reduce code. Something like:

`talentRouter.route('/talent')
.all( commonMiddleware )
.get( specificMiddleware, (req, res, next) => {
  // code
})
.post( specificMiddleware, (req, res, next) => {
  // more code
});
`